src/components/Article.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Article({
+  title,
+  text,
+  date,
+}) {
+  return (
+    <>
+      <h1>
+        {`${title}`}
+      </h1>
+      <p>
+        {`${text}`}
+      </p>
+      <span>
+        {`${date}`}
```

Comment:
```suggestion
        {date}
```
should work like this